docker-compose.yml

```diff
@@ -2,26 +2,30 @@ notaryserver:
   build: .
   dockerfile: server.Dockerfile
   links:
-   - notarymysql
-   - notarysigner
-  ports:
-   - "8080"
-   - "4443:4443"
+    - notarymysql
+    - notarysigner
   environment:
-   - SERVICE_NAME=notary
-  command: -config=fixtures/server-config.json
+    - SERVICE_NAME=notaryserver
+  ports:
+    - "8080"
+    - "4443:4443"
+  entrypoint: /bin/bash
+  command: -c "./migrations/migrate.sh; notary-server -config=fixtures/server-config.json"
```

Comment:
Hmmmm, good question. The migration script currently can't fail. It'll loop until the user kills it. I should print an error message on each loop. I could build in a long failure time (like 30 seconds) after which it can fail the startup.